src/components/ChallengeEditor/index.js

```diff
@@ -1100,6 +1103,7 @@ class ChallengeEditor extends Component {
     if (_.isEmpty(challenge)) {
       return <div>Error loading challenge</div>
     }
+    const isTaskType = checkTaskType(challenge.type)
```

Comment:
@dashu-baba inside this render method we already have variable `isTask` which detect task slightly different, but actually it gives the same result. Let's reuse it inside this method and don't add duplicate `isTaskType` here.

Reply:
@maxceem , Thanks for the update, I will update them accordingly.

Reply:
@maxceem , Could you please share how the request payload should be during creation of a challenge with **reviewerType** & **reviewer**.

Currently in code I can see **reviewerType** is passed  as a prop of **legacy** object. 
I tried to add reviewer to that object. Also, I tried to add reviewType, reviewer in the main object body.
But each time API returning with message "reviewer" is not allowed".

Please provide some support here.

Reply:
I have replied on the ticket https://github.com/topcoder-platform/work-manager/issues/1007